ui/src/main/res/values/strings.xml

```diff
@@ -257,4 +257,14 @@
     <string name="biometric_prompt_private_key_title">Authenticate to view private key</string>
     <string name="biometric_auth_error">Authentication failure</string>
     <string name="biometric_auth_error_reason">Authentication failure: %s</string>
+    <string name="wifi_settings">Wi-Fi Settings</string>
+    <string name="wifi_auto_disconnect_checkbox">Disconnect when using to one of these networks:</string>
+    <string name="auto_disconnect_networks_hint">Network1, Network2, Network3</string>
+    <string name="auto_disconnect_networks_label">Networks</string>
+    <string name="blacklisted_wifi_networks">Blacklisted Wi-Fi networks</string>
```

Comment:
I personally have no issue with it, but the name `blacklist` might offend some people these days (just like `master` branches etc), so it might be good to use a different term..